seed/static/seed/partials/data_upload_modal.html

```diff
@@ -225,7 +225,7 @@ <h4 class="modal-title" ng-switch-when="13" translate>Portfolio Manager Import</
         <div class="row" style="padding-bottom:10px">
             <label class="control-label col-lg-4 col-sm-4" for="inputPMReportTemplateName">Report Template Name <small>{$ pm_templates.length ? '(' + pm_templates.length + ')' : '' $}</small></label>
             <div class="col-lg-8 col-sm-8">
-                <select ng-model="pm_template" ng-disabled="!pm_buttons_enabled" ng-options="x.name for x in pm_templates"></select>
+                <select ng-model="pm_template" ng-disabled="!pm_buttons_enabled" ng-options="x.display_name for x in pm_templates"></select>
```

Comment:
To show the "child" template rows better, I added a display name key to the template dictionaries.  I needed to keep the "name" key unaltered to use it in operations, but this display_name key allows me to indent children objects.  It looks really good.